src/views/account-detail/index.js

```diff
@@ -678,6 +672,7 @@ class AccountLatest extends Component {
                                 name="reveal-pk-passphrase"
                                 id="reveal-pk-passphrase"
                                 placeholder="Passphrase"
+                                onInput={(e) => this.setState({ passphraseInput: e.currentTarget.value})}
```

Comment:
Minor fix, prior to this we were using some jQuery-ish way of retrieving this input's value.